data/language/en-GB.txt

```diff
@@ -3651,6 +3651,15 @@ STR_6396    :Disable screensaver and monitor power saving
 STR_6397    :{SMALLFONT}{BLACK}If checked, screensaver and other monitor power saving features will be inhibited while OpenRCT2 is running.
 STR_6398    :File contains unsupported ride types. Please update to a newer version of OpenRCT2.
 STR_6399    :OpenRCT2 needs files from the original RollerCoaster Tycoon 2 in order to work. Please set the “game_path” variable in config.ini to the directory where you installed RollerCoaster Tycoon 2, then restart OpenRCT2.
+STR_6400    :I have the GOG installer for RollerCoaster Tycoon 2 downloaded, but it is not installed
```

Comment:
```suggestion
STR_6400    :I have the GOG offline installer for RollerCoaster Tycoon 2 downloaded, but it is not installed
```
People might otherwise confuse this with the GOG Galaxy downloader stub. (Of course, plenty still will, but hopefully this will prevent _some_ people from making that mistake.)